README.md

```diff
@@ -81,7 +81,7 @@ const results = await search.query('configuration', {
 });
 ```
 
-- `path` — String of a path in the format of `/foo/bar/`. Results with a path matching or subornate will appear first.
+- `path` — String of a path in the format of `/foo/bar/`. Results with a path matching or subordinate will appear first.
```

Comment:
This word is misspelled in two different ways, but I'm assuming this is the right correction.